docs/core/getting_started/install.md

```diff
@@ -4,29 +4,33 @@
 
 Prefect requires Python 3.7+. If you're new to Python, we recommend installing the [Anaconda distribution](https://www.anaconda.com/distribution/).
 
-To install Prefect, run:
+::: tip 
+Note with the general availability of [Prefect 2.0](https://docs.prefect.io/getting-started/installation/), you will have to specify the Prefect version when installing or updating Prefect 1.0 via a package manager.
+:::
+
+To install Prefect 1.0, run:
 
 :::: tabs
 ::: tab Pip
 
 ```bash
-pip install prefect
+pip install "prefect==1.*"
 ```
 
 :::
 
 ::: tab Conda
 
 ```bash
-conda install -c conda-forge prefect
+conda install -c conda-forge "prefect>=1.2.4,<2"
```

Comment:
Why is the `>=` necessary?

Reply:
I don't think we want to have a specific lower pin

Reply:
What should this line be?